polaris/polaris/utils.py

```diff
@@ -268,7 +260,7 @@ def submit_stellar_deposit(transaction, multisig=False) -> bool:
             "Stellar transaction failed when submitted to horizon: "
             f"{transaction_result.result.results}"
         )
-
+    transaction.claimable_balance_id = get_balanceid(response) if claimable else None
```

Comment:
If a transaction is `pending_trust`, will the transaction's envelope always include a claimable balance op? Or can it have a payment op? If a transaction can be `pending_trust` and use a payment, `get_balance_id()` will return None, which I guess isn't a problem per se, but a developer looking at this probably believes `get_balance_id()` always returns something.